src/qt/rpcconsole.cpp

```diff
@@ -616,6 +616,7 @@ void RPCConsole::setClientModel(ClientModel *model, int bestblock_height, int64_
         ui->peerWidget->setColumnWidth(PeerTableModel::Subversion, SUBVERSION_COLUMN_WIDTH);
         ui->peerWidget->setColumnWidth(PeerTableModel::Ping, PING_COLUMN_WIDTH);
         ui->peerWidget->horizontalHeader()->setStretchLastSection(true);
+        ui->peerWidget->horizontalScrollBar()->setStyleSheet("QScrollBar {height:0px;}");
```

Comment:
Have you tried `ui->peerWidget->setHorizontalScrollBarPolicy(Qt::ScrollBarAlwaysOff)`?

Reply:
It's a NACK from me on removing the horizontal scroll bar from the Peers Table. The user can be left in a state where it is inconvenient to scroll right to see the rest of the content. This is a UX downgrade.

Note: if there's consensus for this change (for some reason), you missed disabling the horizontal scroll bar on the banned peers table

![Screen Shot 2021-05-12 at 8 37 04 PM](https://user-images.githubusercontent.com/23396902/118061228-8a191a80-b362-11eb-81ab-44faac1baf60.png)